test/verify/check-journal

```diff
@@ -471,41 +479,117 @@ s.send(b"PRIORITY=3\\nFOO=bar\\n")'
         sel = "#journal-box .cockpit-logline .cockpit-log-message:contains('crashed in __nanosleep()')"
         b.click(sel)
 
-        # Wait until loaded (when delete button is loaded, all is loaded)
-        sel = "#journal-entry-fields .nav .btn-danger"
-        b.wait_visible(sel)
+        sel = "table.reporting-table tr:first-child .btn:contains('Report')"
+        b.click(sel)
+
+        sel = "table.reporting-table tr:first-child a[href$='/reports/bthash/123deadbeef'"
+        b.wait_present(sel)
+
+        # FIXME: Remove this once reportd-0.7.1 lands there.
+        if m.image != "fedora-31":
+            # "Unreport" the problem to test reporting unknown problem
+            m.execute('find /var/spool/abrt -name "reported_to" -or -name "ureports_counter" | xargs rm')
+            # The service also needs to be restarted, because the daemon maintains
+            # its own cache that interferes with resetting the state.
+            m.execute('systemctl restart reportd')
+            # We have no network access.
+            m.execute("sed -i 's|abrt-action-perform-ccpp-analysis|true|' /etc/libreport/events.d/ccpp_event.conf")
+            # reporter-bugzilla will not be run without a duphash file present.
+            m.execute("find /var/spool/abrt -depth -maxdepth 1 | head -1 | xargs -I {} cp '{}/uuid' '{}/duphash'")
+
+            b.reload()
+            b.enter_page("/system/logs")
+
+            sel = "table.reporting-table tr:nth-child(2) button:contains('Report')"
+            b.click(sel)
+
+            test_user = 'correcthorsebatterystaple'
```

Comment:
:+1: This is being used both as user name and password? It's just against the mock, and I suppose that just ignores it?

Reply:
Yes and yes. Thought I would use something less tired, but didn’t quite feel like using unique username and password.